test/test_math_ops.cpp

```diff
@@ -853,6 +853,52 @@ TEST(MathOps, SquaredDifferenceBroadcasting) {
   opexecuter.RunTest();
 }  // end of test op SquaredDifferenceBroadcasting
 
+// ArgMax test for positive dimension
+TEST(MathOps, ArgMax_Pos) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 2;
+  int dim2 = 3;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
+  AssignInputValuesRandom(A);
+
+  vector<int64> dim = {1};
```

Comment:
We don't need dims to be a vector as you have verified that the TF op takes a single dimension. Please make dim a scalar int.
Would be good to have a test setting the "output_type" attribute to INT_32. Please modify any of the test to add that.